client/index.html

```diff
@@ -1,11 +1,116 @@
+          10.5k
+        </a>
+        <a class="Badge" href="https://twitter.com/Browserslist">
+          <img src="./assets/twitter_logo.svg" alt="">
+          4k
+        </a>
+      </div>
+      <p class="Introduction__about">Shared browser and platform compatibility config for popular JavaScript tools,
+        including
+        Autoprefixer, Babel, ESLint, PostCSS, and Webpack</p>
+      <p class="Introduction__supported">
+        Supported by
+        <img src="./assets/evil_maritians_logo.svg" role="presentation">
+        <a href="" class="Link">Evil Martians</a> and
+        <img src="./assets/cube_logo.svg" role="presentation">
+        <a href="" class="Link">Cube</a>
+      </p>
+    </section>
+    <h2>How to get started</h2>
+    <p>
+      <a data-query="defaults"><code>defaults</code></a> - if you're building a web application for the global audience.
+    </p>
+    <p>
+      <a data-query="node 18"><code>node 18</code></a> - if you're building a Node.js application, e.g., for SSR.
+    </p>
+    <p><a href="" class="Link">Autoprefixer</a> Autoprefixer ,
+      <a href="" class="Link">Babel</a> or any of the following libraries will find target browsers automatically, when you add the
+      following to
+      <b>package.json</b>:
+    </p>
+    <a data-query="defaults, not IE 11, maintained node versions"><pre class="Pre">
+      <code>"browserslist": [</code>
+      <code>  "defaults",</code>
+      <code>  "not IE 11",</code>
+      <code>  "maintained node versions"</code>
+      <code>]</code>
+    </pre></a>
+    <p>You can also check publicly Browserslist configs on <a href="" class="Link">GitHub</a></p>
+
+    <h2>Query Syntax</h2>
+    <p> You can specify the browser and Node.js versions by queries (case insensitive):</p>
+    <ul></ul>
+    <li>
+      <p><a data-query="defaults"><code>defaults</code></a>: Browserslist's default browsers</p>
+      <p><a data-query="> 0.5%"><code>> 0.5%</code></a> , last 2 versions, Firefox ESR, not dead</p>
+    </li>
+    <li>
+      <p>By usage statistics</p>
+      <p>
+        <code>> 5%</code>: browsers versions selected by global usage statistics.
+      </p>
+    </li>
+
+  </article>
+  <article class="Article Article--right">
+    <h2>Display compatible browsers</h2>
+    <form id="query_form" class="Form">
+      <textarea name="query" cols="30" rows="6" required
+                class="Form__queryTextArea" id="query_text_area"
+                placeholder="Write the Browserslist config here..."></textarea>
+      <p class="Form__interactivePlaceholder">Write the Browserslist config here... <br>
+        For example
+        <a data-query="defaults"><code>defaults</code></a> ,
+        <a data-query="not dead"><code>not dead</code></a>
+      </p>
+      <p class="Form__hint Form__hint--progress">Press Enter
+        <span role="presentation" class="Form__enterIcon">
+          <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none">
```

Comment:
Let’s move SVG files to separated files and include them.

If they will be in separated SVG files, we will be able to use graphical editors and `svgo`

Reply:
But you can also create a separated issue for that and do it another PR.

Reply:
There should be an inline svg, because of different color in light and dark theme. This way is better, than have two .svg files for each theme. And this is already optiized by [svg](https://jakearchibald.github.io/svgomg/).

Reply:
Of course, I mean move it to separated file and inline to HTML template by builder.

Something like I did for OKLCH picker https://github.com/evilmartians/oklch-picker/blob/main/view/main/index.pug#L14